helm-build-push/action.yaml

```diff
@@ -0,0 +1,44 @@
+name: Helm build and push chart
+description: Push helm chart to OCI-based registries
+inputs:
+  CHARTS_PATH:
+    description: Path to charts base folder
+    default: ./charts
+    required: false
+  CHART_NAME:
+    description: "Chart to build and push"
+    required: true
+  REGISTRY:
+    description: Registry
+    default: ghcr.io
+    required: true
+  REGISTRY_URL:
+    description: Registry url
+    default: oci://ghcr.io/greenbone/helm-charts/
```

Comment:
```suggestion
    default: oci://${{ vars.IMAGE_REGISTRY }}/greenbone/helm-charts/
```

Reply:
You cannot use repository vars in actions